builder/builder.proto

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018 Google LLC
```

Comment:
Are these types derived directly from a GA4GH spec or no?  If they are, please link to the spec.  If they aren't please document.

Reply:
Added package comment.